src/Hosting/Hosting/src/Internal/HostingMetrics.cs

```diff
@@ -84,33 +88,28 @@ public void RequestEnd(string protocol, bool isHttps, string scheme, string meth
         }
     }
 
-    public void UnhandledRequest()
-    {
-        _unhandledRequestsCounter.Add(1);
-    }
-
     public void Dispose()
     {
         _meter.Dispose();
     }
 
-    public bool IsEnabled() => _currentRequestsCounter.Enabled || _requestDuration.Enabled || _unhandledRequestsCounter.Enabled;
+    public bool IsEnabled() => _activeRequestsCounter.Enabled || _requestDuration.Enabled;
 
     private static void InitializeRequestTags(ref TagList tags, bool isHttps, string scheme, string method, HostString host)
     {
-        tags.Add("scheme", scheme);
-        tags.Add("method", method);
+        tags.Add("url.scheme", scheme);
+        tags.Add("http.request.method", ResolveHttpMethod(method));
         if (host.HasValue)
         {
-            tags.Add("host", host.Host);
+            tags.Add("server.address", host.Host);
```

Comment:
`server.address` and `port` come from the host header which in general case can be anything, so low cardinality is not guaranteed.

Because of this, otel made them opt-in for servers. 
Opt-in means that they are not reported by default, but instrumentation MAY provide a config option to enable them.

We also expect that in common case `server.address : port` are static for one service instance, so users don't get much information from them and OTel resource attributes (service.name, service.instance.id) provide enough information.


So, .NET can
- drop them completely
- add config option to enable them
- report them by default (violating the spec) - the risk here is that if a malicious/buggy client would be able to reach service with a random host header, it would cause cardinality explosion and dimension will be randomly capped. I.e. corresponding metrics will become useless.

Reply:
server.address is required - https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/http-metrics.md#metric-httpserverduration

Reply:
it's an old version, semconv have been moved to https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/http-metrics.md

Reply:
Ok. I think we may drop them for .NET 8. _Might_ be able to add configuring for this release. We'll see.

Reply:
I'd be fine if we drop them. It seems pretty benign to go from 'not supported' to 'available with opt-in config' in a future release based on customer feedback.